platform/admin/configurator/admin-configurator-api/src/main/java/org/codice/ddf/admin/configurator/Status.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.configurator;
+
+/**
+ * <b> This code is experimental. While this class is functional and tested, it may change or be
+ * removed in a future version of the library. </b>
+ */
+public enum Status {
+    COMMIT_PASSED, COMMIT_FAILED, SKIPPED, ROLLBACK_PASSED, ROLLBACK_FAILED;
```

Comment:
An operation would get to commit failed in the following example:
Given [op1, op2, op3, op4]
1. op1 commits successfully
2. op2 commits successfully
3. op3 fails
4. op2 rolls back successfully
5. op1 fails to rollback
the final results would be:
[op1: ROLLBACK_FAILED, op2: ROLLBACK_PASSED, op3: COMMIT_FAILED, op4: SKIPPED]

Any operation that fails to commit will go to the COMMIT_FAILED state and remain there; any subsequent operations will be marked as SKIPPED; all prior operations will be unrolled and rollback will be attempted. Rollback may then pass or fail on each of those.